### PR TITLE
added NM codes 

### PR DESCRIPTION
**Pull request by Firojpaudel**
aru ni kk ho chai tha vayena hai ...
pako jati updated. 👍

### CODE REVIEW

File: NM/back_diff.c
Comment (st0rm47) [inline NM/back_diff.c:1]: Backward Differential

File: NM/divided_difference.c
Comment (st0rm47) [inline NM/divided_difference.c:1]: Divided Difference

File: NM/simpsons1_3.c
Comment (st0rm47) [inline NM/simpsons1_3.c:1]: Simpson's 1/3

File: NM/tabulated_difference.c
Comment (st0rm47) [inline NM/tabulated_difference.c:1]: Tabulated Difference

File: NM/newton_forward.c
Comment (st0rm47) [inline NM/newton_forward.c:1]: No difference at all

File: NM/linear_regression.c
Comment (st0rm47) [inline NM/linear_regression.c:1]: Yo chainxa ra?

File: NM/lagrange.c [hunk 1]
```diff
@@ -1,3 +1,4 @@
+// Lagrange Interpolation
 #include <stdio.h>
 #include <math.h>
 #define MAX 15
```
Comment (st0rm47) [inline NM/lagrange.c]: Complete xaina

**Review by st0rm47 — CHANGES_REQUESTED** [review]

File: NM/linear_regression.c
Comment (Firojpaudel) [reply NM/linear_regression.c:1]: ah re

Comment (Firojpaudel) [reply NM/linear_regression.c:1]: prasan le halya thyo

File: NM/lagrange.c (hunk 1 above)
Comment (Firojpaudel) [reply NM/lagrange.c]: timrai code haina ra yo ??

File: NM/linear_regression.c
Comment (st0rm47) [reply NM/linear_regression.c:1]: thik xa tesobhaye

File: NM/lagrange.c (hunk 1 above)
Comment (st0rm47) [reply NM/lagrange.c]: meroma xa ra, malai thah xaina💀

**Review by st0rm47 — COMMENTED** [review]
DSA commits hatauna paryoo

**Review by st0rm47 — APPROVED** [review]

**Pull request closed without merging**